src/EditorFeatures/CSharp/CodeCleanup/CSharpCodeCleanupService.cs

```diff
@@ -16,7 +16,8 @@ namespace Microsoft.CodeAnalysis.CSharp.CodeCleanup;
 [ExportLanguageService(typeof(ICodeCleanupService), LanguageNames.CSharp), Shared]
 [method: ImportingConstructor]
 [method: Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
-internal sealed class CSharpCodeCleanupService(ICodeFixService codeFixService, IDiagnosticAnalyzerService diagnosticAnalyzerService) : AbstractCodeCleanupService(codeFixService, diagnosticAnalyzerService)
+internal sealed class CSharpCodeCleanupService(ICodeFixService codeFixService)
+    : AbstractCodeCleanupService(codeFixService)
```

Comment:
most changes are just not passing the IDiagAnalzyerService along.  INstead, code that needs it, requests it as a normal workspace service at the point they actually need it.